### ISSUE TITLE
Add hint that colorfont is currently only working in Firefox

### ISSUE THREAD

**Issue by b-g** [hfg-gmuend/openmoji-website#49]
add hint "beta-version currently Firefox only"

![2019-10-23 at 13 31](https://user-images.githubusercontent.com/480224/67418826-8879dd00-f599-11e9-8eb3-df432aa61a5c.jpg)

https://github.com/hfg-gmuend/openmoji/issues/53#issuecomment-545551902 cc @AstroGD

**Comment by felixhaeberle** [comment]
I'll add a support link pointing to this resource: https://www.colorfonts.wtf/ - they outline which apps and browsers are already capable of Opentype SVG.

**Comment by b-g** [comment]
@felixhaeberle No please just ad a hint like "beta-version currently Firefox only" ... our colorfont is quite buggy and only in a proof of concept state. We support even less what is state on colorfonts.wtf :(

**Comment by b-g** [comment]
@felixhaeberle Could you add the hint ""alpha version, currently Firefox only, don't use for production!" in the next days? We get a lot questions on the colorfont in openmoji repo. Would be good to state how bad the colorfont is more explicitly.

**Comment by felixhaeberle** [comment]
@b-g Yeah for sure! Had some busy weeks. Will do this change soon and also the other ones.

**Comment by felixhaeberle** [comment]
@b-g Do you agree with this?

<img width="302" alt="Screenshot 2019-12-04 14 07 13" src="https://user-images.githubusercontent.com/34959078/70145008-6ba7ed80-169f-11ea-8c73-4ca07e8f08b0.png">

**Comment by b-g** [comment]
Hi @felixhaeberle, thanks! Almost!

```
OpenMoji-Black.ttf:
alpha version
```
```
OpenMoji-Color.ttf:
alpha version, currently Firefox only, see [discussion](https://github.com/hfg-gmuend/openmoji/issues/93), don't use for production!
```
Note the link to: https://github.com/hfg-gmuend/openmoji/issues/93

**Comment by felixhaeberle** [comment]
updated this in #58

**Comment by felixhaeberle** [comment]
This can be closed as of #58 🚢
